backend/pkg/runtime_scanner/scanner.go

```diff
@@ -176,8 +179,16 @@ func (s *RuntimeScanner) startScan(scanConfig *ScanConfig) error {
 		namespaces = []string{corev1.NamespaceAll}
 	}
 
+	var parallelism int
+	// If set specific parallelism from web UI, use that
+	if scanConfig.Parallelism != 0 {
+		parallelism = scanConfig.Parallelism
+	} else {
+		parallelism = s.defaultScanParallelism
+	}
+
```

Comment:
`defaultScanParallelism` should be removed from `RuntimeScanner` and `scanConfig.Parallelism` should be set directly to `runtime_scan_config.ScanConfig.MaxScanParallelism`.
I don't think we even need the `Parallelism` on `scanConfig` since we don't have an API support, but if you decided to keep it, please put a TODO to handle it from the UI.